primitives/io/src/lib.rs

```diff
@@ -1443,6 +1447,19 @@ pub trait Offchain {
 	}
 }
 
+/// Interface for querying the local node's mixnet keystore.
+#[runtime_interface]
+pub trait MixnetKxPublicStore {
```

Comment:
This could be moved to e.g.`primitives/mixnet/runtime-api`.

Reply:
AFAICT all the other runtime->host interfaces are defined in this file? `primitives/mixnet/runtime-api` has the host->runtime interface.

Reply:
I see what you mean now. Done this.